src/main/java/uk/gov/justice/digital/delius/transformers/AssessmentTransformer.java

```diff
@@ -10,33 +10,34 @@
 
 public class AssessmentTransformer {
 
-    public static OffenderAssessments assessmentsOf(Offender offender, Optional<OGRSAssessment> OGRSAssessment, Optional<OASYSAssessment> OASYSAssessment) {
+    public static OffenderAssessments assessmentsOf(Offender offender, Optional<OGRSAssessment> ogrsAssessment, Optional<OASYSAssessment> OASYSAssessment) {
         return OffenderAssessments
             .builder()
             .rsrScore(offender.getDynamicRsrScore())
-            .ogrsScore(getOGRSScore(OGRSAssessment, OASYSAssessment))
+            .ogrsScore(getOGRSScore(ogrsAssessment, OASYSAssessment))
+            .orgsLastUpdate(ogrsAssessment.map(OGRSAssessment::getLastUpdatedDate).orElse(null))
             .build();
     }
 
-    private static Integer getOGRSScore(Optional<OGRSAssessment> OGRSAssessment, Optional<OASYSAssessment> OASYSAssessment) {
+    private static Integer getOGRSScore(Optional<OGRSAssessment> oGRSAssessment, Optional<OASYSAssessment> oASYSAssessment) {
```

Comment:
Adding a comment just to see if I can but also, shouldn't it be ogrsAssessment rather than oGRSAssessment

Reply:
Yep , good point , I have renamed them all so that they might look at least consistent to each other